.github/workflows/ci.yml

```diff
@@ -0,0 +1,61 @@
+on:
+  pull_request:
+    branches:
+      - main
+
+name: CI
+
+jobs:
+  build-test:
+    name: Build and test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        rust:
+          - stable
+          - 1.73.0 # MSRV
+        features:
+          - all
+          - debug,default
+          - debug,esplora
+          - debug,snap
+          - debug,bitcoind
+    steps:
+      - name: checkout
+        uses: actions/checkout@v4
+      - name: Install Rust toolchain
+        uses: dtolnay/rust-toolchain@4f647fc679bcd3b11499ccb42104547c83dabe96
+        with:
+          toolchain: ${{ matrix.rust }}
+      - name: Install wasm-pack
+        run: curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh -s -- -f
```

Comment:
What about installing it with `cargo`? Looks like we can using this?

```bash
cargo install wasm-pack
```

I generally try to avoid running shell scripts this way "just in case" 😅

> That's mentionned [here](https://rustwasm.github.io/wasm-pack/installer/) in the "alternative instructions".

Reply:
true! I copied this from the tutorial without thinking.

Reply:
updated with right link linked to a commit hash